src/Build/Resources/Strings.resx

```diff
@@ -931,6 +931,18 @@
     <value>MSB4125: The project file name "{0}" is invalid. {1}</value>
     <comment>{StrBegin="MSB4125: "}UE: The solution filename is provided separately to loggers.</comment>
   </data>
+  <data name="RequiredPropertyNotSetErrorMessagingIsolationMode" xml:space="preserve">
+    <value>
+      MSB4047: The "{0}" task was not given a value for the required parameter "{1}".
```

Comment:
Is this the right error code and message?

Reply:
Addressed [here](https://github.com/dotnet/msbuild/pull/8249#issuecomment-1358019754).